packages/kbn-text-based-editor/src/text_based_languages_editor.tsx

```diff
@@ -466,7 +470,7 @@ export const TextBasedLanguagesEditor = memo(function TextBasedLanguagesEditor({
 
   useEffect(() => {
     const validateQuery = async () => {
-      if (editorModel?.current) {
+      if (editor1?.current) {
```

Comment:
ℹ️ This is irrelevant with this feature but using the model here instead of the editor was creating an error in the console. (worker proxy not found). This is fixing it